x/twap/types/keys.go

```diff
@@ -29,9 +29,16 @@ var (
 	historicalTWAPTimeIndexNoSeparator = "historical_time_index"
 	historicalTWAPPoolIndexNoSeparator = "historical_pool_index"
 
-	mostRecentTWAPsPrefix         = mostRecentTWAPsNoSeparator + KeySeparator
-	historicalTWAPTimeIndexPrefix = historicalTWAPTimeIndexNoSeparator + KeySeparator
-	historicalTWAPPoolIndexPrefix = historicalTWAPPoolIndexNoSeparator + KeySeparator
+	mostRecentTWAPsPrefix = mostRecentTWAPsNoSeparator + KeySeparator
+	// keySeparatorPlusOne is used for creating prefixes for the key end in iterators
```

Comment:
What's the reason we can't use the conventional way of iterating through all stores using prefix stores?